source/tutorials/write-update-documents.txt

```diff
@@ -101,42 +94,43 @@ To learn more about update operators, see
 Update One Document
 -------------------
 
-To update a single document from a MongoDB collection, call the ``updateOne()``
+To update a single document in a MongoDB collection, call the ``updateOne()``
 method and pass your query filter as the first argument. Then, pass your
-update operators as the second argument to the ``updateOne()`` method. After you call the
-``updateOne()`` method, chain the ``subscribe()`` method and pass a subscriber
-as an argument.
+update operators as the second argument to the ``updateOne()`` method. Finally,
+pass the ``updateOne()`` method to the static ``Mono.from()`` method from the
+``Mono Publisher``. In {+java-rs+}, you must use ``Publisher`` implementations to control how data
+is transmitted in your application. The ``Mono Publisher`` is a ``Publisher`` implementation from the Project Reactor library. To learn more
```

Comment:
S: I think it might be better to swap these two sentences so that the explanation of `Mono` comes right after its first mention. 

```suggestion
``Mono Publisher``. The ``Mono Publisher`` is a ``Publisher`` implementation from the Project Reactor library. In {+java-rs+}, you must use ``Publisher`` implementations to control how data
is transmitted in your application. To learn more
```